contrib/masterfiles-stage/README.org

```diff
@@ -101,6 +105,15 @@ when developing new staging methods.
     - 88335d36b48c8808b12b48667a463182dc8d0338
     - mytag
 
+- channel_config_file - The path to the config file for multiple policy channels
+  - Example value:
+    - /var/cfengine/policy_channels/channel_to_source.txt
```

Comment:
Interesting idea.  I think yes, it could.  (It would be in params, not options, as it's VCS_TYPE specific.)  I think simplest would be just one array channel_config which would be expected to contain at least two elements, and if it had an odd number of elements the last would be ignored (perhaps with a warning).